mgmt/mgmt-core/src/main/java/deployment/mgmt/atrifacts/strategies/classpathfile/ArtifactClasspathReader.java

```diff
@@ -0,0 +1,42 @@
+package deployment.mgmt.atrifacts.strategies.classpathfile;
+
+import deployment.mgmt.atrifacts.Artifact;
+
+import java.io.File;
+import java.util.List;
+import java.util.Objects;
+
+import static deployment.mgmt.utils.ZipUtils.readInnerFile;
+import static java.util.Arrays.stream;
+import static java.util.stream.Collectors.toList;
+
+public class ArtifactClasspathReader implements JarClasspathReader {
```

Comment:
please, write unit test

Reply:
can you please explain how to do this?

Reply:
add manually prepared jar file to test resources, add pass reference to it to the class

Reply:
added test